homeassistant/components/onvif/camera.py

```diff
@@ -111,6 +112,24 @@
         DOMAIN, SERVICE_PTZ, async_handle_ptz, schema=SERVICE_PTZ_SCHEMA
     )
 
+    async def async_handle_reboot(service):
+        """Handle PTZ service call."""
+        all_cameras = hass.data[ONVIF_DATA][ENTITIES]
+        entity_ids = await async_extract_entity_ids(hass, service)
+        target_cameras = []
+        if not entity_ids:
```

Comment:
We are dropping this logic for all other integrations in #29178. Let's not use it here.